spec/views/documents/history/_whitehall_entry.html.erb_spec.rb

```diff
@@ -20,4 +20,55 @@
     expect(rendered).to have_content(I18n.t!("documents.history.by") + " John Smith")
     expect(rendered).to have_content(I18n.t!("documents.history.entry_types.whitehall_migration.new_edition"))
   end
+
+  it "shows an internal note imported timeline entry with an author" do
```

Comment:
Since we have tests for a timeline entry with an author, and there's no special behaviour for displaying the author when it comes to internal notes, fact check requests or fact check responses, I don't think we need to test for these again.  We do however have logic for what we show for each of these timeline entry types, so it would be good to test for these.  For example, we should have a test that the body of the internal note is displayed.

Reply:
@emmabeynon Thanks for your comments and suggestions on this.  I've updated the tests as appropriate and added new tests for the highlighting.